### ISSUE TITLE
What about search api?

### ISSUE THREAD

**Issue by arturdev** [SuperZombi/HdRezkaApi#5]
Are you planning to add an API for the Search feature?

**Comment by SuperZombi** [comment]
This feature is currently not planned. Sorry

**Comment by shizukushq** [comment]
@arturdev
You can use their official API to search. `/engine/ajax/search.php`
```python
async def search(data: Query):
    rezka_domain = "hdrezka.ag"
    rezka_url = f"https://{rezka_domain}/engine/ajax/search.php"
    rezka_data = {
        'q': data.q,
    }

    async with ClientSession() as session:
        async with session.post(rezka_url, data=rezka_data, ssl=False) as response:
            if response.status == 200:
                text = await response.text()
                soup = BeautifulSoup(text, 'html.parser')
                movies = []

                for item in soup.select('.b-search__section_list li'):
                    title = item.find('span', class_='enty').text
                    url = item.find('a')['href']

                    rating_span = item.find('span', class_='rating')
                    rating = rating_span.find('i').text if rating_span else None

                    year_match = re.search(r'-(\d{4}).', url)
                    year = year_match.group(1) if year_match else None

                    movie = {
                        'title': title,
                        'url': url,
                        'rating': rating,
                        'year': year,
                    }
                    movies.append(movie)

                return movies
            else:
                return {'status': 'error', 'message': 'Failed to fetch data'}, 500

```
change `rezka_domain` to a domain available in your country

**Comment by SuperZombi** [comment]
Implemented in version `7.6.0`

**Comment by SuperZombi** [comment]
Advanced search (by pages) in `8.0.0`
